app/models/user.rb

```diff
@@ -0,0 +1,13 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  # Validations for the User model
+  validates :name, presence: true
+  validates :posts_count, numericality: { greater_than_or_equal_to: 0 }
+
+  def recent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] Kindly add tests for the validations and the recent_posts method.


![model](https://github.com/kayjeee/Kayjeee_Postgresql_blogapp/assets/53124548/794a6077-f473-4f69-aa69-f90a580dad91)